plugin/dadbod.vim

```diff
@@ -18,6 +18,8 @@ call extend(g:db_adapters, {
 
 call extend(g:, {'db_adapter_mongodb#srv': 'db#adapter#mongodb#'}, 'keep')
 
+call extend(g:, {'db#adapter#redis': 'db#adapter#rediss'}, 'keep')
```

Comment:
This is backwards. And you should be using it _instead_ of `rediss.vim`, not in addition to it.

Reply:
Ah, that makes a lot more sense, really appreciate the clarity, let me see if I can get that working.

Reply:
@tpope pushed an update, let me know if this better maps your expectations.